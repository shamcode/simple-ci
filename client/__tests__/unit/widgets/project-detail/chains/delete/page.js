import Page from '../../../../../../src/widgets/project-detail/chains/delete/page.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( Page, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
