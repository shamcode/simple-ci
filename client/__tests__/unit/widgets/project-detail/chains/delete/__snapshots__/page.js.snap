// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "page",
  "Options": Object {
    "ID": "widget",
    "containerSelector": "body",
    "types": Array [],
  },
  "html": "
<!--BreadCrumb-->
<!--if-->
",
}
`;
